policy/DecamMapper.paf

```diff
@@ -755,4 +755,12 @@ datasets: {
         persistable:   "Config"
         storage:       "ConfigStorage"
     }
+    singleFrameDriver_config: {
+        template:      "config/singleFrameDriver.py"
+        python:        "lsst.pipe.drivers.singleFrameDriver.SingleFrameDriverConfig"
+        persistable:      "Config"
+        storage:    "ConfigStorage"
+        tables:        raw
+        tables:        raw_visit
+    }
```

Comment:
Strictly, I don't believe it does, as processCcd is doing all the saving etc, but it should probably be in there anyway incase it is expanded later.